src/server/pkg/serviceenv/config.go

```diff
@@ -65,6 +65,8 @@ type PachdSpecificConfiguration struct {
 	DeploymentID               string `env:"CLUSTER_DEPLOYMENT_ID,default="`
 	RequireCriticalServersOnly bool   `env:"REQUIRE_CRITICAL_SERVERS_ONLY",default=false"`
 	MetricsEndpoint            string `env:"METRICS_ENDPOINT",default="`
+	// TODO: Merge this with the worker specific pod name (PPS_POD_NAME) into a global configuration pod name.
```

Comment:
Also lame, but I want to keep these changes from rippling out into a larger set of changes. I think this is okay for the time being.